orchestra/utils/view_helpers.py

```diff
@@ -1,6 +1,7 @@
 from rest_framework import permissions
```

Comment:
could you come up with a manual testing checklist and verify it?

Reply:
Testing checklist:

1. Select a project task. Assign its project to todos project. http://dev2.b12.io:8260/orchestra/app/task/71171
2. Check off a todo. Refresh. Make sure the state is saved
3. Uncheck a todo. Refresh. Make sure the state is saved
4. QA: mark, comment, copy, paste

Do the same after switching back to the `main` branch.

Branch: `task-unification-todo-viewset-reuse-serializer`
https://www.loom.com/share/a6c8df6276974c30aae3ecff5df16023

Branch: `main`
https://www.loom.com/share/0c4e4430128f44e2882e31bfbbdbb80b

Reply:
The fact that I had to perform the first step of the testing checklist means, that we should not merge this PR into the main branch. Otherwise, we should revert the filtering logic to `queryset.filter(task__project__id=project_id)`